src/platform/backends/hyperv_api/hcs_virtual_machine.h

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (C) Canonical, Ltd.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; version 3.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#pragma once
+
+#include <hyperv_api/hcs/hyperv_hcs_compute_system_state.h>
+#include <hyperv_api/hcs/hyperv_hcs_system_handle.h>
+
+#include <multipass/signal.h>
+#include <multipass/virtual_machine_description.h>
+#include <shared/base_virtual_machine.h>
```

Comment:
Nitpick: this is closer to this file than generic multipass includes. Following the idea of including closely related headers first, this should come right after hyperv_api ones.